test/tla/Bug2750.cfg

```diff
@@ -0,0 +1,9 @@
+\* Add statements after this line.
+SPECIFICATION Spec
+CONSTANTS 
+    Producers = {p1,p2}
+    Consumers = {c1,c2}
+    BufCapacity = 1
+\* generate dot file: add "-dump dot trans.dot" to TLC options
+\* (=> extension settings)
```

Comment:
I was confused at first, how this is relevant for the regression test.
Then I realized it's not. Can we remove it?
```suggestion
```

Reply:
I just copied the files as they were submitted in #2750, and only renamed them. We can axe this if you like.